src/types.ts

```diff
@@ -179,3 +179,27 @@ export interface RawTechnicalQuestion {
   follow_ups: string;
   source: string;
 }
+
+export type ReportType = "interview" | "question";
+
+export type ReportStatus = "pending" | "resolving" | "completed";
+
+export interface ReportBody {
+  associated_id: string;
+  reporter_user_id: number;
+  type: ReportType;
+  reason: string;
+}
+
+export interface RawReport {
+  admin_id: null | string;
```

Comment:
nit: undefined instead of null

Reply:
I would just do adminId?: string; instead of declaring undefined.